### PR TITLE
Cortx 27956 new: Implement action event interface

### PR DESCRIPTION
**Pull request by gauri-bhosale**
# Problem Statement
- Add const.py file under fault_tolerance directory with the content as mentioned in the LLD.
Add "Health Status Event Message Schema" with the default values populated as mentioned in the LLD
https://jts.seagate.com/browse/CORTX-27956

# Design:
Create Event Message Schema Class with initialize default value.
https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/891781277/Action+Status+Update+LLD?focusedCommentId=909705441#Health-Status-Event-Message-Schema

# Coding
-  [X] Coding conventions are followed and code is consistent

# Testing :

Event object __dict__ : 
{'event': {'header': {'version': '1.0', 'timestamp': '1645446906', 'event_id': '16454469064844cfc1ff3f451ca36de8fa1c4385a7'}, 'payload': {'source': None, 'cluster_id': None, 'site_id': None, 'rack_id': None, 'storageset_id': None, 'node_id': None, 'resource_type': None, 'resource_id': None, 'resource_status': None, 'specific_info': {}}}}

payload from producers
{'rack_id': 5, 'node_id': 6, 'specific_info': {'data': '1'}}

Changed event object : event.__dict__ : 

{'event': {'header': {'version': '1.0', 'timestamp': '1645446906', 'event_id': '16454469064844cfc1ff3f451ca36de8fa1c4385a7'}, 'payload': {'source': None, 'cluster_id': None, 'site_id': None, 'rack_id': 5, 'storageset_id': None, 'node_id': 6, 'resource_type': None, 'resource_id': None, 'resource_status': None, 'specific_info': {'data': '1'}}}}

# Review Checklist 
- [X ] PR is self reviewed
- [X] JIRA number/GitHub Issue added to PR
- [X]Jira and state/status is updated and JIRA is updated with PR link
- [X] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [X] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

**Review by indrajitzagade — APPROVED** [review]

**Comment by ArchanaLimaye** [comment]
Please fix codacy issue

**Review by Madhura-08 — APPROVED** [review]
Please fix codacy issue

File: ha/fault_tolerance/event.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+VERSION = "1.0"
+
+class Event:
+    """
+    Class for health status event message schema,
+    where producer can import this class object and add it
+    in there existing schema.
+    """
+
+    def __init__(self):
```
[the remark this answers is not in the document]
Comment (Madhura-08) [reply ha/fault_tolerance/event.py:31]: Yes.
https://jts.seagate.com/browse/CORTX-28915

File: ha/fault_tolerance/event.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            payload
+
+    def set_payload(self, _input):
+        """
+        Update payload values.
+        params: payload data of attributes with value type : dict
+        _input = {"source" : "hare", "node_id" : "2"}
+        """
+        for key in _input.keys():
+            if key in list(
```
Comment (ArchanaLimaye) [inline ha/fault_tolerance/event.py:65]: We can log a warning if key is not in list 
However its low priority and can be done when the code is moved to utils

**Review by ArchanaLimaye — APPROVED** [review]

**Review by shaileshsvc — APPROVED** [review]

File: ha/fault_tolerance/event.py (hunk 2 above)
Comment (gauri-bhosale) [reply ha/fault_tolerance/event.py:65]: Will handle this in ticket https://jts.seagate.com/browse/CORTX-28915 thanks @ArchanaLimaye

**Review by shaileshsvc — APPROVED** [review]
Approved

**Review by ArchanaLimaye — APPROVED** [review]

**Comment by indrajitzagade** [comment]
We have ignored one doc string related codacy issue since it was conflicting with another codacy rule.

**Pull request merged**